.github/workflows/publish-container.yaml

```diff
@@ -16,41 +16,62 @@ jobs:
     permissions:
       contents: read
       packages: write
+    env:
```

Comment:
You don't need this at the top level; only need it for the tagging step

Reply:
higher visibility here, and also how we do it everywhere else